INFO:root:Contract /../data/reentrancy/solidity_contract/14284.sol:Halo3D:
INFO:symExec:Running, please wait...
Traceback (most recent call last):
  File "/root/osiris/taintFlow.py", line 652, in perform_taint_analysis
    propagate_taint(taint, tainted_stack, tainted_memory, tainted_storage, instruction, current_stack, previous_block, current_block, next_blocks, arithmetic_errors, sha3_list, false_positives, strings)
  File "/root/osiris/taintFlow.py", line 425, in propagate_taint
    if arithmetic_error["instruction"].data_in[0] == 32 or arithmetic_error["instruction"].data_in[1] == 32:
  File "/root/osiris/z3/z3.py", line 302, in __nonzero__
    return self.__bool__()
  File "/root/osiris/z3/z3.py", line 307, in __bool__
    elif is_false(self):
  File "/root/osiris/z3/z3.py", line 1380, in is_false
    return is_app_of(a, Z3_OP_FALSE)
  File "/root/osiris/z3/z3.py", line 1164, in is_app_of
    return is_app(a) and a.decl().kind() == k
  File "/root/osiris/z3/z3.py", line 885, in decl
    return FuncDeclRef(Z3_get_app_decl(self.ctx_ref(), self.as_ast()), self.ctx)
  File "/root/osiris/z3/z3.py", line 280, in __init__
    Z3_inc_ref(self.ctx.ref(), self.as_ast())
  File "/root/osiris/z3/z3.py", line 630, in as_ast
    return Z3_func_decl_to_ast(self.ctx_ref(), self.ast)
  File "/root/osiris/z3/z3core.py", line 2441, in Z3_func_decl_to_ast
    r = _elems.f(a0, a1)
ArgumentError: argument 2: <type 'exceptions.Exception'>: timeout
INFO:symExec:	============ Results ===========
INFO:symExec:	  EVM code coverage: 	 10.2%
INFO:symExec:	  Arithmetic bugs: 	 True
INFO:symExec:	  └> Overflow bugs: 	 True
/../data/reentrancy/solidity_contract/14284.sol:Halo3D:695:30
((tokenPriceIncremental_)**2)*(tokenSupply_**2)
^
/../data/reentrancy/solidity_contract/14284.sol:Halo3D:691:29
(_tokenPriceInitial**2)
^
/../data/reentrancy/solidity_contract/14284.sol:Halo3D:691:29
(_tokenPriceInitial**2)
^
/../data/reentrancy/solidity_contract/14284.sol:Halo3D:697:30
2*(tokenPriceIncremental_)*_tokenPriceInitial*tokenSupply_
^
/../data/reentrancy/solidity_contract/14284.sol:Halo3D:73:18
ambassadorAccumulatedQuota_[_customerAddress] + _amountOfEthereum
^
INFO:symExec:	  └> Underflow bugs: 	 True
/../data/reentrancy/solidity_contract/14284.sol:Halo3D:67:34
totalEthereumBalance() - _amountOfEthereum
^
INFO:symExec:	  └> Division bugs: 	 False
INFO:symExec:	  └> Modulo bugs:   	 False
INFO:symExec:	  └> Truncation bugs: 	 False
INFO:symExec:	  └> Signedness bugs: 	 False
INFO:symExec:	  Callstack bug: 	 False
INFO:symExec:	  Concurrency bug: 	 False
INFO:symExec:	  Time dependency bug: 	 False
INFO:symExec:	  Reentrancy bug: 	 False
INFO:symExec:	 --- 49.9948801994 seconds ---
INFO:symExec:	====== Analysis Completed ======
INFO:root:Contract /../data/reentrancy/solidity_contract/14284.sol:SafeMath:
INFO:symExec:Running, please wait...
WARNING: invalid function application for if, sort mismatch on argument at position 1, expected Bool but given #x0000000000000000000000000000000000000000000000000000000000000001